Source/Core/DolphinQt2/DPI.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/DPI.h"
+
+#include <QGuiApplication>
+#include <QPaintDevice>
+#include <QPixmap>
+#include <QScreen>
+#include <QSize>
+#include <QString>
+#include <algorithm>
+
+double DPI::GetScale()
```

Comment:
Noticed this now as well, but any particular reason these functions aren't enclosed in the namespace via a block like in the header?